crates/indexer/src/models/token_models/ans_lookup.rs

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use std::collections::HashMap;
+
+use crate::{
+    schema::current_ans_lookup,
+    util::{bigdecimal_to_u64, parse_timestamp_secs},
+};
+use aptos_api_types::{deserialize_from_string, Transaction as APITransaction};
+use bigdecimal::BigDecimal;
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+
+pub const ADDR: &str = "0xdbf606fea404cb26efe68d00f8f4fff8e4b9ce69f903818f8acf81473a32430a";
```

Comment:
idk if we want this to be a config or what, but this will be different depending on the network

Reply:
Oh...Hmm...this is annoying.

Reply:
I mean it'll have to be a config then...